pkg/volume/util/metrics.go

```diff
@@ -94,7 +94,7 @@ func OperationCompleteHook(plugin, operationName string) func(types.CompleteFunc
 		timeTaken := time.Since(requestTime).Seconds()
 		// Create metric with operation name and plugin name
 		status := statusSuccess
-		if *c.Err != nil {
+		if c.Err != nil {
```

Comment:
This comparison is not nil safe so I fixed it

Reply:
What would it mean for c.Err to be non-nil, but c.Err to be nil?  This would change how we handle that case

Reply:
Hmm, I cannot tell why they choose to use `*error` instead of `error` since I'm not confident with this change I'll remove this possible breaking change!

Reply:
@dashpole I've updated it so would you review the PR again? 🙏